src/styles/common/_buttons.scss

```diff
@@ -1,35 +1,87 @@
+@import '../constants/typography';
+@import '../constants/layout';
+@import '../mixins/buttons';
+
 /**************************************
  * Buttons
  **************************************/
+$bdl-btn-border-width: 1px;
+$bdl-btn-height: 32px;
+$bdl-btn-height-large: 40px;
+$bdl-btn-icon-size: 20px;
+$bdl-btn-text-icon-size: 16px;
+$bdl-btn-padding-horizontal: $bdl-grid-unit * 4;
+$bdl-btn-padding-vertical: 10px; // Do not reuse, use flexbox or line-height instead
```

Comment:
If we don't want some of these to be reused, should we move them inside of `.btn` to keep them from being global variables?